pom.xml

```diff
@@ -89,19 +89,19 @@
         <maven.compiler.source>1.8</maven.compiler.source>
 
         <!-- DEPENDENCY CODE VERSIONS       -->
-        <test-containers.version>1.17.2</test-containers.version>
-        <spotbugs.version>4.0.3</spotbugs.version>
+        <test-containers.version>1.17.3</test-containers.version>
+        <spotbugs.version>4.7.2</spotbugs.version>
         <log4j.version>2.17.1</log4j.version>
         <docker-java.version>3.2.13</docker-java.version>
-        <kafka.version>3.0.0</kafka.version>
+        <kafka.version>3.2.3</kafka.version>
         <slf4j.version>1.7.32</slf4j.version>
-        <fasterxml.jackson-core.version>2.11.3</fasterxml.jackson-core.version>
+        <fasterxml.jackson-core.version>2.12.6</fasterxml.jackson-core.version>
```

Comment:
Why ot go to 2.13.x?

Reply:
I wanted to be in sync with the `operator` repository.